models/experimental/functional_yolov4/reference/downsample3.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+import torch
+import torch.nn as nn
+from models.experimental.functional_yolov4.reference.resblock import ResBlock
+
+
+class DownSample3(nn.Module):
```

Comment:
please generate the pytorch graph of DownSample3 from here and compare to the downSample3 of the reference graph from the original implementation attached here. Also include the auto-generated TTNN graph. and lets ensure all 3 graphs match as a sanity check.

Reply:
![downsample3_ref](https://github.com/tenstorrent-metal/tt-metal/assets/164333733/faf38b49-e572-44b8-b5f0-f983cb21e65b)